docs/source/getting-started/authenticators-users-basics.md

```diff
@@ -1,5 +1,7 @@
 # Authentication and User Basics
 
+(authenticators-users-basics)=
```

Comment:
```
/home/runner/work/jupyterhub/jupyterhub/docs/source/getting-started/config-basics.md:81: WARNING: more than one target found for 'myst' cross-reference authenticators-users-basics: could be :std:ref:`authenticators` or :std:doc:`authenticators` [myst.ref]
```

This warning means that the reference `authenticators-users-basics` is already in use in another file, `config-basics.md`. So maybe we could change this reference to `authenticators-getting-started`?

Reply:
Sorry, I changed it to just `authenticators` before seeing this comment. Can that work? I see all tests now pass?

Reply:
Yes, so long as it is a unique reference, it will work :)